### ISSUE TITLE
Compilation with antlr3 3.5.2

### ISSUE THREAD

**Issue by nuqleo** [Linphone-sync/belle-sip#4]
Hi,

Current antlr3 version in Fedora is 3.5.2 but belle-sip 1.4.0 compilation failed because it supports antlr3 3.2 and 3.4. See compilation log https://nucleo.fedorapeople.org/belle-sip.log

Please, add possibility to compile belle-sip with antlr3 3.5.2.

**Comment by cicku** [comment]
I'd like to know why it doesn't support antlr3.5, any reasons here?

(Is github just a place for synchronization?)

**Comment by nuqleo** [comment]
Looks like just git.linphone.org synchronized here, but Linphone don't have any other bug trucking system.
They have only mailing list but emails often lost there without answers
https://lists.nongnu.org/archive/html/linphone-developers/2015-01/msg00002.html

**Comment by cicku** [comment]
Yes, so I think it's better to build antlr 3.4 in belle-sip package, you don't have any other choices now as patching the software to antlr3.5 may be awkward.
